code-of-conduct.html

```diff
@@ -0,0 +1,134 @@
+
+</p>If you still feel threatened our team members will be more than happy to help you contact venue security/local law enforcement or otherwise assist you to feel safe for the remainder of the event. We value your participation and your security is of great importance.</p>
+
+
+
+<h2>Internal version:</h2>
+
+
+<h3>Warnings:</h3>
+
+<p>A member of staff can issue a verbal warning to a participant if their behaviour violates the Code of Conduct. Warnings should also be forwarded to the relevant organiser as soon as is practical. The report should include:</p>
+
+<ul>
+	<li>Identifying information of victim and perpetrator.</li>
+	<li>Time the warning was issued.</li>
+	<li>Actions deemed in violation of the Code of conduct.</li>
+	<li>Approximate time of the violation (if different from the time of warning).</li>
+	<li>Identifying information of other parties involved (eg. if assistance from fellow staff was required/if victim was accompanied by another eye witness).</li>
+</ul>
+
+<h3>Presentations</h3>
+
+<p>Presentations should not be stopped for a one-time assumable error, though a staff member should speak to the presenter afterward. In the event that the code of conduct is continuously/seriously disregarded, staff should take immediate action to politely and professionally stop the presentation as soon as deemed necessary. For example, by simply saying “I apologize for the disruption but this presentation cannot be continued at this time.”</p>
+
+
+<h3>Taking Reports</h3>
+
+<ul>
+	<li>Record what is being said.</li>
+	<li>Assure them they are being taken seriously and the violating action is intolerable.</li>
+	<li>Ask for the relevant info listed in the above sub heading titled “Warnings” but not push for information if they seem visibly uncomfortable or otherwise reluctant.</li> 
+	<li>All reports must be recorded and passed along to the appropriate person.</li>
+	<li>If requested by the reporter further action may be taken (eg. contact of organiser/law enforcement).</li>
+	<li>Do not pressure the reporter to take further action if they don’t want to but also mention means of action that can be taken (such as escort etc).</li>
+	<li>Don’t share unnecessary details of the report with others.</li>
+</ul>
+
+<h3>Expulsion</h3>
+
+<p>Reasons for possible expulsion:</p>
+
+<ul>
+	<li>A series of offences resulting in a warning from staff.</li>
+	<li>Continuation of harassment after being instructed to stop.</li>
+	<li>A single serious offense (eg. punching/groping).</li>
+</ul>
+
+<p>Consult with other staff members regarding incidents when possible but also act when necessary.<p>
```

Comment:
Again, probably convenient to add another button linking back to the main page here to save people having to scroll back up or navigate by other means:
```
<p>Consult with other staff members regarding incidents when possible but also act when necessary.<p>
<a class="btn btn-primary pull-right" href="https://capetown2018.satrdays.org">Back to main page</a>
</div>
</body>
```